tests/blob/apis/blockblob.test.ts

```diff
@@ -179,23 +181,19 @@ describe("BlockBlobAPIs", () => {
   });
 
   it("upload should fail when metadata names are invalid C# identifiers @loki @sql", async () => {
-    let invalidNames = [
-      "1invalid",
-      "invalid.name",
-      "invalid-name",
-    ]
+    let invalidNames = ["1invalid", "invalid.name", "invalid-name"];
     for (let i = 0; i < invalidNames.length; i++) {
       const metadata = {
         [invalidNames[i]]: "value"
       };
       let hasError = false;
       try {
-        await blockBlobClient.upload('b', 1, {
+        await blockBlobClient.upload("b", 1, {
           metadata: metadata
         });
       } catch (error) {
         assert.deepStrictEqual(error.statusCode, 400);
-        assert.strictEqual(error.code, 'InvalidMetadata');
+        assert.strictEqual(error.code, "InvalidMetadata");
```

Comment:
Could we separate the format change to a single commit at begin, so we can just skip this commit in code review.

Reply:
Sure. It'll take a bit to undo but I'm happy to do that after I test against a real storage account to figure out what to tweak